Watchman.DomainModel/Warns/Queries/Handlers/GetWarnEventsQueryHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using Watchman.Cqrs;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Warns.Queries.Handlers
+{
+    public class GetWarnEventsQueryHandler : IQueryHandler<GetWarnEventsQuery, GetWarnEventsQueryResults>
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public GetWarnEventsQueryHandler(ISessionFactory sessionFactory)
+        {
+            this._sessionFactory = sessionFactory;
+        }
+
+        public GetWarnEventsQueryResults Handle(GetWarnEventsQuery query)
+        {
+            using var session = this._sessionFactory.Create();
+            var filteredEvents = session.Get<WarnEvent>().Where(x => x.ReceiverId == query.UserId);
+            if (query.ServerId != 0)
```

Comment:
Sugestia bardziej dla całego repo, nie warto w takim przypadku dodać w zmienną 0 w jakimś const?
Oczywiście można się domyślić, ale jest to trochę magic number
@Marcin99b @gagyn

Reply:
Stała ta jest w GetAllMessagesQuery bo wcześniej była używana tylko w obrębie messages
myślę, że w tym przypadku najlepiej dać ServerId jako nullable i sprawdzać query.ServerId.HasValue

Reply:
tylko że w sytuacjach gdzie będzie tam obiekt
to mamy dodatkową instancje za każdym razem 

nullable lepiej używać do sytuacji, gdzie 0 ma konkretne znaczenie
i jest wyraźna różnica między 0 i null

Reply:
nie ruszyłem bo nie wiem na czym stanęło, mogę zrobić obojętnie jak chcecie

Reply:
@Marcin99b chyba tak jak jest?

Reply:
no powinno być normalne a nie nullable

Reply:
ok to zostawiam